<ui version="4.0" >
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>347</width>
    <height>352</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>347</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MidasClient 2.0 User Registration</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="DionysusIcons.qrc" >
    <normaloff>:/icons/document_add.png</normaloff>:/icons/document_add.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupbox" >
     <property name="font" >
      <font>
       <underline>true</underline>
      </font>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="3" >
       <widget class="QTextEdit" name="textEdit" >
        <property name="font" >
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="html" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:8pt;">&lt;span style=" font-size:14pt; font-weight:600; text-decoration: underline;">Disclaimer :&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:14pt;">MIDAS Client Agreement&lt;br />Version 1.0, June 1, 2007&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:14pt;"> &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:14pt;">Copyright (c) Kitware Inc. All rights reserved. See &lt;a href="http://www.kitware.com/Copyright.htm">&lt;span style=" text-decoration: underline; color:#0000ff;">http://www.kitware.com/Copyright.htm&lt;/span>&lt;/a> for details.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:14pt;">&lt;/p>
&lt;p align="justify" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg'; font-size:14pt;">This software is distributed WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the above copyright notices for more information.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>141</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="agreeButton" >
        <property name="font" >
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="text" >
         <string>I Agree</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QRadioButton" name="disagreeButton" >
        <property name="font" >
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="text" >
         <string>I Disagree</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="nameLabel" >
     <property name="text" >
      <string>First Name :</string>
     </property>
     <property name="buddy" >
      <cstring>firstNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="firstNameEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="parentLabel" >
     <property name="text" >
      <string>Last Name :</string>
     </property>
     <property name="buddy" >
      <cstring>lastNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lastNameEdit" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="nameLabel_4" >
     <property name="text" >
      <string>Affiliation :</string>
     </property>
     <property name="buddy" >
      <cstring>firstNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="affiliationEdit" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="nameLabel_2" >
       <property name="text" >
        <string>Email :</string>
       </property>
       <property name="buddy" >
        <cstring>firstNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="emailEdit" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="nameLabel_3" >
       <property name="text" >
        <string>Retype Email :</string>
       </property>
       <property name="buddy" >
        <cstring>firstNameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="retypeEmailEdit" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="DionysusIcons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>336</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
